src/app/newSettings/Sections/Workspace/Logs/AccessLogsSection.tsx

```diff
@@ -0,0 +1,255 @@
+import { useState } from 'react';
+import { Table, TableBody, TableCell, TableHeader, TableRow } from '@internxt/ui';
+import { ArrowDown, ArrowUp } from '@phosphor-icons/react';
+
+import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
+import Section from '../../../../newSettings/components/Section';
+import { ScrollableTable } from 'app/shared/tables/ScrollableTable';
+import { AccessLogsFilterOptions } from './components/AccessLogsFilterOptions';
+import { useAccessLogs } from './hooks/useAccessLogs';
+import { WorkspaceLogPlatform, WorkspaceLogType } from '@internxt/sdk/dist/workspaces';
+import dateService from 'app/core/services/date.service';
+import { useDebounce } from 'hooks/useDebounce';
+import { getEnumKey } from '../../../utils/LogsUtils';
+import { LoadingRowSkeleton } from 'app/shared/tables/LoadingSkeleton';
+
+interface LogsViewProps {
+  onClosePreferences: () => void;
+}
+
+interface HeaderItemsProps {
+  title: string;
+  isSortByAvailable: boolean;
+  width: string;
+  sortKey?: 'updatedAt' | 'platform' | 'type';
+  defaultSort?: 'ASC' | 'DESC';
+}
+
+export const AccessLogsSection = ({ onClosePreferences }: LogsViewProps): JSX.Element => {
+  const { translate } = useTranslationContext();
+  const [searchMembersInputValue, setSearchMembersInputValue] = useState<string>('');
+  const [daysFilter, setDaysFilter] = useState<number | undefined>();
+  const [activityFilter, setActivityFilter] = useState<WorkspaceLogType[]>([]);
+  const [orderBy, setOrderBy] = useState<{ key: 'updatedAt' | 'type' | 'platform'; direction: 'ASC' | 'DESC' }>({
+    key: 'updatedAt',
+    direction: 'DESC',
+  });
+  const debouncedSearchMemberValue = useDebounce(searchMembersInputValue, 500);
+  const { accessLogs, isLoading, hasMoreItems, loadMoreItems } = useAccessLogs({
+    activity: activityFilter,
+    lastDays: daysFilter,
+    member: debouncedSearchMemberValue,
+    orderBy: [orderBy.key, orderBy.direction].join(':'),
+  });
+
+  function getActivityType(type: WorkspaceLogType) {
+    const getLogKey = getEnumKey(WorkspaceLogType, type);
+    return translate(`preferences.workspace.accessLogs.filterActions.activity.${getLogKey}`) || 'Unknown action';
+  }
+
+  function getPlatformType(platform: WorkspaceLogPlatform) {
+    const getPlatformKey = getEnumKey(WorkspaceLogPlatform, platform);
+    return translate(`preferences.workspace.accessLogs.filterActions.platform.${getPlatformKey}`) || 'Unknown';
+  }
```

Comment:
Use same code style to maintain consistency, these functions are written with conventional style and the others with arrow style